src/main/java/com/sourcegraph/cody/SubscriptionPanel.kt

```diff
@@ -4,11 +4,52 @@ import com.intellij.ide.BrowserUtil
 import com.intellij.ide.ui.laf.darcula.ui.DarculaButtonUI
 import com.intellij.ui.dsl.builder.panel
 import com.sourcegraph.common.CodyBundle
+import com.sourcegraph.common.UpgradeToCodyProNotification
 import com.sourcegraph.config.ConfigUtil
+import com.sourcegraph.config.ThemeUtil
 
 fun createSubscriptionTab(isCurrentUserPro: Boolean) = panel {
+  val chatLimitError = UpgradeToCodyProNotification.chatRateLimitError.get()
+  val autocompleteLimitError = UpgradeToCodyProNotification.autocompleteRateLimitError.get()
+  val chatLimitLabelValue =
+      "You used all ${chatLimitError?.limit} chat messages for this month. Upgrade to Cody Pro to get unlimited chats."
```

Comment:
are we sure we want to display that value here? I am asking in the context of https://github.com/sourcegraph/jetbrains/pull/274

Reply:
I guess we need to wait for reply from @taylorsperry